src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drm.c

```diff
@@ -537,6 +542,14 @@ static struct drm_driver mm_drm_driver = {
 	.date				= driver_date,
 };
 
+const struct drm_gem_object_funcs xocl_gem_object_funcs = {
+        .free = xocl_free_object,
+        .get_sg_table = xocl_gem_prime_get_sg_table,
+        .vmap = xocl_gem_prime_vmap,
+        .vunmap = xocl_gem_prime_vunmap,
+        .vm_ops = &xocl_vm_ops,
```

Comment:
gem_prime_export is missing from this ops. Please check.

Reply:
@chvamshi-xilinx made changes as suggested. Thanks